packages/lisk-transactions/test/1_register_second_passphrase.ts

```diff
@@ -135,7 +137,7 @@ describe('#registerSecondPassphrase transaction', () => {
 		it('should have a signSignature property when passphrase is provided', () => {
 			return expect(
 				registerSecondPassphraseTransaction.signSignature,
-			).not.to.be.eql(null);
+			).to.be.eql(signSignature);
```

Comment:
this test should be reverted.
and in SecondSignatureTransaction, it should override the `sign` function not to assign signSignature

Reply:
Done. Resolving.